lib/src/analyzers/lint_analyzer/anti_patterns/anti_patterns_list/long_method.dart

```diff
@@ -37,6 +41,10 @@ class LongMethod extends ObsoletePattern {
     final issues = <Issue>[];
 
     for (final function in functions) {
```

Comment:
I'd suggest to use where instead: `for (final function in functions.where(_isExcluded).toList())`